cssreloader.background.js

```diff
@@ -2,7 +2,7 @@
 
     var storageKey = "shortcutOptions";
     var defaultShortcutOptions = {
-        "keyIdentifier"      : 'F9',
+        "key"                : 'F9',
```

Comment:
Let's keep the same storage key in order to avoid exciting users having their settings reset.